### ISSUE TITLE
Migrated VLAN filter is not valid in the admin

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#5430]
**Describe the bug**
Migrated VLAN filter is not valid in the admin (not sure if backend or frontend fault)

**To Reproduce**
1. VLAN filter before migration:
```
[category_empty]
filter = node_info.category
operator = not_defined
value =

[99_set_default_category:category_empty]
scope = RegisteredRole
role = normal
```
2. Run upgrade script
3. Resulting filter:
```
[99_set_default_category]
status=enabled
condition=not_defined(node_info.category, "")
role=normal
scopes=RegisteredRole
```
4. Error in the admin:
![image](https://user-images.githubusercontent.com/3857942/81570554-c04a4580-936e-11ea-9ae1-9b8fa2d9df6f.png)

**Comment by julsemaan** [comment]
@jrouzierinverse, can you confirm if the migration script created the expected result and the error would be in the frontend

**Comment by jrouzierinverse** [comment]
This was fixed by 92d6d5a50a86411393a3f2ce256e4561127a7f47

**Comment by jrouzierinverse** [comment]
And it is already in maintenance
